apple/macbook-pro/12-1/default.nix

```diff
@@ -7,6 +7,21 @@
     <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix>
   ];
 
+  powerManagement = {
+    # enable gradually increasing/decreasing CPU freq, rather than using "powersave", which would keep CPU freq at 0.8Ghz.
```

Comment:
Can you explain how this behavior is better than the current default?

Reply:
for me when using default governor "powersave" did not scale CPU freq at all, and I had extremely low performance, almost unusable in gui applications.

Reply:
Thanks for explanation.